### PR TITLE
test: Adds tests to the CssEditor component

### PR DESCRIPTION
**Pull request by michael-s-molina**
### SUMMARY
Adds tests to the `CssEditor` component.

### TEST PLAN
1 - Execute `CssEditor` tests
2 - All tests should pass

@rusackas @junlincc 

### ADDITIONAL INFORMATION
- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Review by yardz — APPROVED** [review]

File: superset-frontend/src/dashboard/components/CssEditor/index.jsx [hunk 1]
```diff
@@ -33,7 +33,6 @@ const propTypes = {
 const defaultProps = {
   initialCss: '',
   onChange: () => {},
-  templates: [],
```
Comment (pkdotson) [inline superset-frontend/src/dashboard/components/CssEditor/index.jsx:36]: I'm curious to why this was removed?

Comment (pkdotson) [inline superset-frontend/src/dashboard/components/CssEditor/index.jsx:36]: I'm curious to why this was removed?

Comment (pkdotson) [reply superset-frontend/src/dashboard/components/CssEditor/index.jsx:36]: actually nvm, i see not needed really.

**Review by pkdotson — APPROVED** [review]
LGTM!

**Pull request merged**